src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -79,6 +79,7 @@
   public static final String SCHEMA_IGNORE_CONFIG = "schema.ignore";
   public static final String TOPIC_SCHEMA_IGNORE_CONFIG = "topic.schema.ignore";
   public static final String DROP_INVALID_MESSAGE_CONFIG = "drop.invalid.message";
+  public static final String ROUTING_FIELD_NAME = "routing.field.name";
```

Comment:
Why is the ROUTING_FIELD_NAME = "routing.field.name" ?
Why is it "routing.field.name" ?
Can it be something else ?
Where is it used ?

Reply:
It is used to configure which field should be used as routing, when we initiate the ES connector, see https://github.com/Everbridge/data-pipeline/pull/8 for usage example.